src/main/java/nextstep/subway/domain/Line.java

```diff
@@ -0,0 +1,110 @@
+package nextstep.subway.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.ForeignKey;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import nextstep.subway.constant.ErrorCode;
+import nextstep.subway.utils.StringUtils;
+
+@Entity
+public class Line extends BaseEntity {
+
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false, unique = true)
+    private String name;
+    @Column(nullable = false)
+    private String color;
+    @ManyToOne
+    @JoinColumn(name = "up_station_id", foreignKey = @ForeignKey(name = "fk_line_to_upstation"), nullable = false)
+    private Station upStation;      // 상행종점역
```

Comment:
```suggestion
    private Station upStation;
```
단순 주석은 사용하지 않는편이 어떨까요?
주석이 없어도 충분한 의미 전달이 가능한 네이밍을 사용하면 되고, 주석을 추가로 관리해줘야되는점에서 여기서는 굳이 사용하지 않아도 될 것 같아요 :)